tests/cannot-write-post.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { expect, test } from "@playwright/test";
+
+
+test("cannot write post because buttons are not rendered", async ({ page }) => {
```

Comment:
buttons are not rendered .... because user is not loggedIn?

Reply:
yes. the buttons are not rendered because the user is not logged in.